schemas/document-generation-completed.go

```diff
@@ -0,0 +1,18 @@
+package schemas
+
+//  DocumentGenerationCompleted struct for document-generation-completed topic
+type DocumentGenerationCompleted struct {
+	RequesterID           string            `avro:"requester_id"                    json:"requester_id"`
+	ID                    string            `avro:"id"                              json:"id"`
+	Description           string            `avro:"description"                     json:"description"`
+	DescriptionIdentifier string            `avro:"description_identifier"          json:"description_identifier"`
+	DocumentCreatedAt     string            `avro:"document_created_at"             json:"document_created_at"`
+	DescriptionValues     DescriptionValues `avro:"description_values"              json:"description_values"`
+	Location              string            `avro:"location"                        json:"location"`
+	DocumentSize          string            `avro:"document_size"                   json:"document_size"`
+}
+
+//  DescriptionValues struct for document-generation-completed topic
```

Comment:
Please use only a single space after the double slash.

Reply:
Remove in [ce005fb](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/ce005fb8926cbc2899d61cf66de233a4e80ce97c)

Reply:
Not done.

Reply:
double spaces removed in [9edd492](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/9edd49223eaff7f92ad875ac4925a15528511c59)